docs/variables.md

```diff
@@ -7,6 +7,7 @@ weight: 15
 # Variable Substitutions Supported by `Tasks` and `Pipelines`
 
 This page documents the variable substitutions supported by `Tasks` and `Pipelines`.
+**Note:** Tekton does not support shell-escaped parameters. As a task author you are responsible for escaping your own parameters.
```

Comment:
The language here reads a bit ambiguous to me - "Tekton does not support shell-escaped parameters" in a variables doc feels a tad out-of-nowhere.  I could understand this language in the params section of the TaskRun doc for the additional context that would provide.

My suggestion here would be to keep the language variables-focused: "Tekton does not escape the contents of variables.  Task authors are responsible for properly escaping a variable's value according to the shell, image or scripting language that the variable will be used in."

WDYT?

Reply:
@sbwsg Yeah, that is way better. I didn't know what to really type. Just wanted to make this public knowledge. Thank you so much!